modules/client/src/interfaces.ts

```diff
@@ -553,20 +555,57 @@ export type SubgraphPluginRepoListItem = {
   releases: SubgraphPluginRepoReleaseListItem[];
 };
 
+export type SubgraphPluginVersion = {
+  build: number;
+  metadata: string;
+  release: {
+    release: number;
+    metadata: string;
+  };
+  pluginRepo: {
+    id: string;
+    subdomain: string;
+  };
+};
+
```

Comment:
That's interesting, but this would probably change the type names we already have. Anyway, the short term suggestion is to split the types we create/export from the types that are given to us by the subgraph model

Reply:
Okay we can make a task to isolate the subgraph types into its own file. I think we should give it a test in the future because I remember that you can modify the output of the generated types so it maches our naming conventions